service/src/main/java/uk/gov/hmcts/reform/fpl/model/Address.java

```diff
@@ -7,17 +7,21 @@
 import lombok.Builder;
 import lombok.Data;
 
+import javax.validation.constraints.NotBlank;
+
 @Data
 @Builder
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Address {
 
+    @NotBlank(message = "Enter a valid address for the contact")
```

Comment:
I’d thought the same. But I guess ‘contact’ is generic enough to work in future situations no? Failing that I could create an address validator. But yet again I think it’s a content problem.

Reply:
I guess it will be grouped into the section with the heading, so that gives some context. There was a chat I had around only validating applicant address, how do we do this?

Reply:
Currently the only case event that validates address is applicant. This is because we have used @valid annotation on the caseData. Other cases events that use children i.e respondents, children will not validate address.